hack/confromance-image-config.yaml

```diff
@@ -0,0 +1 @@
+dockerLibraryRegistry: mirror.gcr.io/library
```

Comment:
Our image proxy would also be able to act as a registry, right @fgimenez? Could we maybe use that when `"${CI}" == "true"`?

Reply:
hmm I don't think so, we set our proxy in the daemon's config and it acts transparently to the clients using it, they keep requesting images from dockerhub, quay, etc. For this to work we need additionally to setup the proxy with a concrete CA and add it as a trusted CA to the hosts where the damons run. All this is explained here https://github.com/rpardini/docker-registry-proxy#configuring-the-docker-clients--kubernetes-nodes--linux-clients

Reply:
I think it should be the default because we can hit docker pull rate limit easily also on a local setup

Reply:
It would be great if we would add a way to inject the proxy settings in CI. Let's do that in a follow-up.